### DISCUSSION TITLE
[Question] CardFormField UI Customization

### DISCUSSION THREAD

**Discussion by manumohn** [flutter-stripe/flutter_stripe#1455] [Q&A]
Hi Folks!

Thanks for all the work on this package. Saved me a ton of time while integrating stripe to my app.

I needed some assistance with customising CardFormField. Is it possible to get the "ScanCard" option and the grouped information layout similar to the PaymentSheet, in CardFormField? 

<details>
  <summary>Attaching images for reference.</summary>

![image](https://github.com/flutter-stripe/flutter_stripe/assets/22583762/a470a8e8-70a6-4330-bcdf-80af13f656af)

![image](https://github.com/flutter-stripe/flutter_stripe/assets/22583762/a0bd4e9b-3ef9-49d9-b429-7b63071ec31e)

</details>

**Comment by remonh87** [comment]
Hi no the scan card is only available in the paymentsheet. I really would recommend this integration since it is simpler more modern and also Stripe is developing this more actively than cardforms.

**Reply by manumohn** [reply]
Thanks for the quick revert @remonh87. 

Im actually trying to save the user's card for a future payment. Would this be possible in the payment bottom sheet?
Additionally, would it be possible to do a payment hold via the payment bottom sheet?
